examples/blog-multiple-authors/src/layouts/post.astro

```diff
@@ -6,7 +6,7 @@ import authorData from '../data/authors.json';
 const { content } = Astro.props;
 ---
 
-<html>
+<html dir="ltr" lang="en">
```

Comment:
Could you pull out the lang to work based on the frontmatter like in https://github.com/snowpackjs/astro/commit/1b13f5c158b5fbe157406308205c0939b415e1d7 ?

Ex:
```astro
const { content } = Astro.props;
---

<html lang={ content.lang ?? 'en' }>
```
^^^ this tries to use the `lang` variable from the frontmatter of the content, if it is undefined, it uses english.

Reply:
Gotcha, that's a great suggestion! Will do that 👍